.github/workflows/release.yml

```diff
@@ -0,0 +1,19 @@
+name: Release
+on:
+  push:
+    branches: [main]
+    tags: ["*"]
```

Comment:
Paranoia/sanity check, is there any scenario given the current GH repo settings where someone could extract these secrets by opening a PR from a fork? (secrets are probably masked in GH actions output but only if matching 1:1... there'd be no problem e.g. base64-encoding them first)

Reply:
(In CircleCI, I think we've got `Pass secrets to builds from forks` disabled)

Reply:
> Secrets are not passed to workflows that are triggered by a pull request from a fork.